admin/index.html

```diff
@@ -1,25 +1,80 @@
-<!doctype html>
+<!DOCTYPE html>
 <html>
-<head>
-  <meta charset="utf-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>ADA Content Manager</title>
-</head>
-<body>
-   <!-- Include the script that builds the page and powers Netlify CMS -->
-  <script src="https://unpkg.com/netlify-cms@^2.0.0/dist/netlify-cms.js"></script>
-   <link href="/admin/config.yml" type="text/yaml" rel="cms-config-url">
-   <style>
-     img[alt="Logo"] {
-       margin-left: 75px;
-       width: 150px;
-     }
-     p {
-       color: red;
-     }
-     button {
-       margin-top: 0px !important;
-     }
-   </style>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>ADA Content Manager</title>
+  </head>
+  <body>
+    <!-- Include the script that builds the page and powers Netlify CMS -->
+    <script src="https://unpkg.com/netlify-cms@^2.0.0/dist/netlify-cms.js"></script>
+    <script>
+      // We need to grab out most current version of our CSS file and our custom JS. How can we do that?
```

Comment:
What if you did:
```
---
layout: null
---
<!doctype html>
```
to tell jekyll to process this file, then you can use asset includes:

```
{% include scripts.html %}
{% asset styles.css %} {% asset vendor/uswds/uswds-init.min.js !type %}
```